tests/v2/validation/deleting/delete_cluster_test.go

```diff
@@ -34,11 +37,35 @@ func (c *ClusterDeleteTestSuite) SetupSuite() {
 }
 
 func (c *ClusterDeleteTestSuite) TestDeletingCluster() {
-	clusterID, err := clusters.GetV1ProvisioningClusterByName(c.client, c.client.RancherConfig.ClusterName)
-	require.NoError(c.T(), err)
+	tests := []struct {
+		name   string
+		client *rancher.Client
+	}{
+		{"cluster", c.client},
+	}
+
+	for _, tt := range tests {
+		clusterID, err := clusters.GetV1ProvisioningClusterByName(c.client, c.client.RancherConfig.ClusterName)
+		require.NoError(c.T(), err)
+
+		cluster, err := tt.client.Steve.SteveType(clusters.ProvisioningSteveResourceType).ByID(clusterID)
+		require.NoError(c.T(), err)
+
+		updatedCluster := new(apisV1.Cluster)
+		err = v1.ConvertToK8sType(cluster, &updatedCluster)
+		require.NoError(c.T(), err)
+
+		if strings.Contains(updatedCluster.Spec.KubernetesVersion, "rke2") {
+			tt.name = "Deleting RKE2 " + tt.name
+		} else {
+			tt.name = "Deleting K3S " + tt.name
+		}
```

Comment:
couldn't you use similar logic for rke1 and remove the rke1 test file?

Reply:
That seems a bit excessive IMO, considering we already have a separate rke1 test file. I don't see why we wouldn't just keep it as it is.